apps/code-examples/src/app/code-examples/datetime/date-range-picker/basic/demo.component.ts

```diff
@@ -33,7 +34,12 @@ function dateRangeExcludesWeekend(
   standalone: true,
   selector: 'app-demo',
   templateUrl: './demo.component.html',
-  imports: [FormsModule, ReactiveFormsModule, SkyDateRangePickerModule],
+  imports: [
+    FormsModule,
+    ReactiveFormsModule,
+    SkyDateRangePickerModule,
+    CommonModule,
```

Comment:
Was the code example not working without `CommonModule`?

Reply:
oop that was a miss from testing something else and i needed the json pipe lol